jellyfin_kodi/monitor.py

```diff
@@ -144,9 +144,8 @@ def onNotification(self, sender, method, data):
             self.void_responder(data, item)
 
         elif method == 'GetServerAddress':
-
-            server_data = server.auth.get_server_info(server.auth.server_id)
-            server_address = server.auth.get_server_address(server_data, server_data['LastConnectionMode'])
+            
```

Comment:
Nit-picky extra new line here.  No idea why it was there in the old version

Reply:
They are all like that so I just kept it the same as the rest